### PR TITLE
Munny lamp

### PR DESCRIPTION
**Pull request by jedgarpark**

### CODE REVIEW

File: Munny_Lamp/Munny_Lamp.ino [hunk 1]
```diff
@@ -0,0 +1,217 @@
...
+  analogWrite(GREEN_LED, 0);
+  analogWrite(BLUE_LED, 255);  // startup color, waiting for BLE connection
+
+  if (! lis.begin(0x18)) {   // change this to 0x19 for alternative i2c address
+    Serial.println("Couldnt start LIS3DH");
+    while (1);
+  }
+
+  Serial.begin(115200);
+  Serial.println(F("Adafruit Bluefruit MUNNY Neopixel Color Picker"));
```
Comment (ladyada) [inline Munny_Lamp/Munny_Lamp.ino]: fix

Comment (jedgarpark) [reply Munny_Lamp/Munny_Lamp.ino]: Oh, is it to fix the name since it isn't the NeoPixel we're using? Got it, thanks.

Comment (ladyada) [reply Munny_Lamp/Munny_Lamp.ino]: yah is not neopixels!

**Comment by jedgarpark** [comment]
OK, fixed up that comment @ladyada

**Pull request merged**